src/v2/layouts/MemberDataGrid/MemberDataGridContainer.tsx

```diff
@@ -38,12 +38,13 @@ export const MemberDataGridContainer = (): JSX.Element => {
 	const [checkIds, setCheckIds] = useState<StudentCorrectionResponse[]>([]);
 	const [isVisibleConfigModal, setIsVisibleConfigModal] = useState<boolean>(false);
 	const { headerConfig } = memberGridMetaData;
+	const [activeStatus, setActiveStatus] = useState('Active');
 
 	useEffect(() => {
 		if (fetchStudentResp?.studentList.length) {
 			setStudentList(fetchStudentResp.studentList);
-		} else {
-			fetchStudents({ secretKey: secretKey });
+		} else if (!isSuccess && !isLoading) {
+			fetchStudents({ secretKey, historyData: 0, isActive: 1 });
```

Comment:
Create an enum or constant map for historyData values and keep the constant in the Api file.
```suggestion
			fetchStudents({ secretKey, historyData: HISTORY_DATA_FLAGS.EXCLUDE, isActive: STUDENT_ACTIVE_STATUS_FLAG.ACTIVE});
```
Example:
const HISTORY_DATA_FLAGS = {
  INCLUDE: 1,
  EXCLUDE: 0

}